pkgs/applications/networking/mailreaders/mmh/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, ncurses, autoreconfHook, flex }:
```

Continuing from @alyssais 's comment,

```suggestion
{ stdenv, fetchurl, ncurses, autoreconfHook, flex }:

```

Personally, I prefer:

```suggestion
{ stdenv
, fetchurl
, ncurses
, autoreconfHook
, flex
}:

```

With this ^, diffs look much better if dependencies are changed.